R/irace_helpers.R

```diff
@@ -0,0 +1,64 @@
+paradoxToIrace = function(ps){
+  assertClass(ps, "ParamSet")
+  # what about ParamUty = vector numeric/real
+  class_lookup = data.frame(paradox = c("ParamLgl","ParamInt","ParamDbl","ParamFct"),
+                            irace = c("c","i","r","c"), stringsAsFactors = FALSE)
+  name <- ps$ids()
+  type <- class_lookup[match(ps$class, class_lookup$paradox), 2]
```

Comment:
data.table probably has a more elegant way to do this

Reply:
do you know the best vectorized alternative to `subset`?

Reply:
or I guess `join` instead

Reply:
I've used merged, not sure it is more elegant but I'll let you decide